doc/ref/copyright.xml

```diff
@@ -66,28 +66,20 @@ make.
 In addition to the general copyright for &GAP; set forth above,
 the following terms apply to the versions of &GAP; for Windows.
 <P/>
-The executable of &GAP; for Windows that we distribute was compiled with
+&GAP; for Windows is packaged with several packages from
+<Package>Cygwin</Package> (<URL>https://cygwin.com</URL>).
+Files from <Package>Cygwin</Package> are provided under their respective
+licenses (included).
+The &GAP; for Windows executables that we distribute are released under
```

Comment:
Note that I've pluralised 'executable' here since we intend to offer 32-bit and 64-bit versions.

Reply:
Fine, and I am happy with the plural anyway (e.g. we may have bat + exe + dll files even for a single 64-bit mode)